incubator/argo-ingress/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for cloudflare-argo-ingress.
+
+# replicaCount is used only when cloudflare loadbalancing is enabled
+replicaCount: 1
+
+image:
+  repository: gcr.io/stackpoint-public/argot
+  tag: 0.5.2
+  pullPolicy: IfNotPresent
+
+## configure the ingressClass value in order to manage different ingress-controller 
+## deployments each for a different set of ingresses
+## if not set, defaults to the value "argo-tunnel"
+# handledIngressClass: argo-tunnel-custom-value
+
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+rbac:
+  install: true
+  serviceAccount: cloudflare-argo
+  roleName: cloudflare-argo
+
+# Application logging level, higher values give more verbose output
+logging:
+  level: 2
+
+## Cloudflare account details
+
+# The secret contains the base64-encoded origin certificate for your cloudflare domain.
+# Set the install flag false to configure manually in the cluster later.
+# See https://developers.cloudflare.com/argo-tunnel/quickstart/quickstart/ for information on
+# obtaining the origin certificate
+secret:
+  install: false
+  domain: example.co
+  certificate_b64: LS0tLS1CRUdJT...insert base64-encoded certificate here
```

Comment:
Should be empty by default

Reply:
Conflicted on whether or not it makes sense to update. The source of the problem is upstream.

Reply:
(we just clone the repo from cloudflare)